FlowCrypt/Resources/en.lproj/Localizable.strings

```diff
@@ -48,6 +48,7 @@
 "keyServiceError_retrieve_error" = "Could not retrieve keys from DataService. Please restart the app and try again.";
 "keyServiceError_retrieve_parse" = "Could not parse keys from storage. Please reinstall the app.";
 "keyServiceError_retrieve_unexpected" = "Could not import key. Please try to relogin.";
+"keyServiceError_missing_current_email" = "Missing current email. Please restart the app and try again.";
```

Comment:
Not sure that user will understand this. It is not possible to get this situation.

Reply:
True, thanks for raising it. Looks ok enough for now, we'll improve it later (I'll file an issue)

Reply:
Related issue for the future https://github.com/FlowCrypt/flowcrypt-ios/issues/863